plugin/src/main/java/net/aufdemrand/denizen/scripts/commands/BukkitCommandRegistry.java

```diff
@@ -4256,7 +4339,17 @@ public void registerCoreMembers() {
         // <w@world.thunder_duration>
         //
         // @Usage
-        // TODO: Document Command Details
+        // Makes the weather sunny
+        // - weather sunny
+        //
+        // @Usage
+        // Makes the weather storm in world "cookies"
+        // - weather storm world:cookies
+        //
+        // @Usage
+        // Make the weather storm for the player "Rickyling"
+        // - weather type:player storm player:p@Rickyling

```

Comment:
Best to just use the generic `<player>`

Reply:
Or rather, just ... drop the `player:` entirely and say that it affects the linked player.